app/views/components/_markdown_editor.html.erb

```diff
@@ -3,6 +3,7 @@
   component_classes = %w[app-c-markdown-editor govuk-form-group]
   heading_buttons ||= false
   blockquote_button ||= false
+  numbered_list_button ||= false
```

Comment:
My initial thought was to have all of the controls turned on by default, but I couldn't get it to work with the default set to "true", the options were always on. That's why I went with this approach of having to turn on the toolbar options you want.

I used the booleans approach because that seemed to follow the existing pattern for components.  So are you suggesting, having say `controls ||= [:headings, :blockquotes]` set here, so in this example, if you don't pass in controls, you'll see the heading and blockquotes buttons, but that if you want to see more, you have to explicitly pass them in? e.g. `controls: [:headings, :blockquotes, :numbered_list]`?

Reply:
Yup, that sounds a good approach to me. Can do that in a separate PR if you like to not block this one?

Reply:
I've got a separate commit ready. Will push that first to check everything is alright before splitting it up and fixing up.